gpcontrib/gpcloud/src/gpreader.cpp

```diff
@@ -33,9 +33,6 @@ unsigned long gpcloud_id_function(void) {
 
 int thread_setup(void) {
     mutex_buf = new pthread_mutex_t[CRYPTO_num_locks()];
```

Comment:
but what if `new` fails to allocate? I think better to change to `new (std::nothrow)`